support-figma/extended-layout-plugin/src/ui.html

```diff
@@ -676,21 +689,42 @@
         const values = Array.from(strArrayElement.getElementsByTagName("item")).map(
           item => item.textContent);
         jsonStr["text"] = values;
-        inputStringData.push(jsonStr);
+        const extras = getDescriptionExtras(strArrayElement);
+        jsonStr["extras"] = extras;
+        inputStringArray.push(jsonStr);
       }
 
-      createInputStringTable("inputStrings", inputStringData);
+      createInputStringTable("inputStrings", inputStringArray);
     })
     reader.readAsText(selectedFile);
     showOutputsButton.textContent = "Next";
   }
 
+  function getDescriptionExtras(strElement, translatable) {
```

Comment:
this `translatable` parameter isn't used.

Reply:
Done.